src/views/Asset.vue

```diff
@@ -523,6 +526,8 @@ export default {
       navAct: "mint",
       info: true,
       tokenSelected: null,
+      showInfo: false,
+      showInfoButtonText: "More Info",
```

Comment:
should keep same aspect unless something is on purpose
```suggestion
      showInfoButtonText: "Gas Info",
```